docs/src/installation/deployment.md

```diff
@@ -574,11 +574,68 @@ If you decided to set up a dex server, you can parametrize the [metal role](http
+
+As you can see, the Management Firewalls are the first bastion hosts in a partition to provide access to our infrastructure. There are two of them in each partition to guarantee high availability and load balancing. The very first configuration of these routers has to be done manually to solve the chicken and egg problem that you need the management firewalls in place to deploy the partition. Manually means that we generate a configuration template with ansible that we deploy with copy/paste, and the load, through the machine console. Once the management server has been deployed, we are able to deploy this configuration via gitlab-runner and ansible. For this you need the user and the ssh-key, which is deployed with the configuration file mentioned above.
+The Edgerouters has to fulfill some requirements including:
+
+- provide and restrict access to the management network from the internet with a firewall ruleset
+- provide destination NAT to the management server and its IPMI interface (public IP access from a specific IP address only)
+- provide onieboot and ztp via dhcp options for the management spine
+- provide dhcp management addresses for management spine, management server and ipmi interface of the management server
+- Hairpin-NAT for the management server to access itself via its puplic IP, needed by the gitlab-runner to delegate CI-Jobs.
+- propagate a default gateway via BGP
 
-Documentation will follow soon. Please come to our Slack channel when you have come to this point and require any help.
+### Management Servers
+
+The second bastion hosts are the management servers. They are the main bootstrapping components of the management network. They also act as jump hosts for all components in a partition. Once they are installed and deployed, we are able to bootstrap all the other components. To bootstrap the management servers, we generate an ISO image which will automatically install an OS and an ansible user with ssh keys. It is preconfigured with a preseed file to allow an unattended OS installation for our needs. This is why we need remote access to the IPMI interface of the Management Servers: The generated ISO is attached via the virtual media function of the BMC. Ater that, all we have to do is boot from that virtual CD-ROM and wait for the installation to finish. Deployment jobs (Gitlab-CI) in a partition are delegated to the appropriate Management Servers, therefore we need a gitlabci-runner active on each management server.
+After the gitlab CI runner has been installed, you can trigger your Playbooks from the Gitlab-CI. The Ansible-Playbooks have to make sure that these functionalities are present on the Management Servers:
+
+- BMC proxy / BMC reverse proxy
+- Prometheus and exporters
+- gitlab-runner
+- bmc-catcher
+- image-cache
+- simple webserver to provide images
+- Onieboot and ZTP for cumulus switches
+- DHCP addresses for ipmi interfaces of the workers
+- DHCP addresses for cumulus switches
```

Comment:
I think, some of these components are already available in metal-roles. How much more can we move there? It would be great to have a playbook example or a pointer to mini-lab where you can look up how really do it.